oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
I agree that the two definitions will begin to float away from one another. [Reading about][1] ways to inherit docstrings now.

[1]: http://stackoverflow.com/questions/2025562/inherit-docstrings-in-python-class-inheritance